test/platform-stats.test.js

```diff
@@ -91,7 +99,7 @@ describe('platform-stats', () => {
           total_measurement_count
         FROM
           daily_stations
-        ORDER BY station_id`
+        ORDER BY station_id, participant_address, inet_group`
```

Comment:
This test is separate from the code changes, but it was randomly failing due to ordering issues